include/GafferImage/OpenColorIOTransform.h

```diff
@@ -50,13 +52,17 @@ class OpenColorIOTransform : public ColorProcessor
 
 	public :
 
-		OpenColorIOTransform( const std::string &name=defaultName<OpenColorIOTransform>() );
+		OpenColorIOTransform( const std::string &name=defaultName<OpenColorIOTransform>(), const bool &withContextPlug=false );
 		virtual ~OpenColorIOTransform();
 
 		/// Fills the vector will the available color spaces,
 		/// as defined by the current OpenColorIO config.
 		static void availableColorSpaces( std::vector<std::string> &colorSpaces );
 
+
+		Gaffer::CompoundDataPlug *contextPlug();
```

Comment:
not sure what would happen if you call this from a derived class that set the withContextPlug to false, should this be defined on derived class even if that means duplication of code or there a better way?